src/js/components/Beacon/BeaconCommon/BeaconQueryFormUi.tsx

```diff
@@ -233,7 +233,7 @@ const BeaconQueryFormUi = ({
             {hasVariants && (
               <Col xs={24} lg={12}>
                 <Card
-                  title={t('entities.Variants')}
+                  title={t('entities.variant', { count: 100 })} // false count – just need the highest form of plural
```

Comment:
Nice didn't know about the plural feature.
Since these objects are passed to the `t` function a bunch all across the app, I think we could use constants instead to make it easier to write and understand. Something like:
```typescript
// in a constants file
export const T_PLURAL_COUNT = { count: 100 };
export const T_SINGULAR_COUNT = { count: 1 };

// usage
title={t('entities.variant', T_PLURAL_COUNT)}
```
I think this way it would be clear even without the comment.

Reply:
good idea!